antlr/parser/grulev3/grulev3_parser.go

```diff
@@ -1283,8 +1223,7 @@ func (p *grulev3Parser) ThenExpressionList() (localctx IThenExpressionListContex
 	p.GetErrorHandler().Sync(p)
 	_la = p.GetTokenStream().LA(1)
 
-
-	for ok := true; ok; ok = (((_la) & -(0x1f+1)) == 0 && ((int64(1) << uint(_la)) & ((int64(1) << grulev3ParserMINUS) | (int64(1) << grulev3ParserTRUE) | (int64(1) << grulev3ParserFALSE) | (int64(1) << grulev3ParserNIL_LITERAL) | (int64(1) << grulev3ParserNEGATION))) != 0) || ((((_la - 38)) & -(0x1f+1)) == 0 && ((int64(1) << uint((_la - 38))) & ((int64(1) << (grulev3ParserSIMPLENAME - 38)) | (int64(1) << (grulev3ParserDQUOTA_STRING - 38)) | (int64(1) << (grulev3ParserSQUOTA_STRING - 38)) | (int64(1) << (grulev3ParserDECIMAL_FLOAT_LIT - 38)) | (int64(1) << (grulev3ParserHEX_FLOAT_LIT - 38)) | (int64(1) << (grulev3ParserDEC_LIT - 38)) | (int64(1) << (grulev3ParserHEX_LIT - 38)) | (int64(1) << (grulev3ParserOCT_LIT - 38)))) != 0) {
+	for ok := true; ok; ok = (((_la)&-(0x1f+1)) == 0 && ((1<<uint(_la))&((1<<grulev3ParserMINUS)|(1<<grulev3ParserTRUE)|(1<<grulev3ParserFALSE)|(1<<grulev3ParserNIL_LITERAL)|(1<<grulev3ParserNEGATION))) != 0) || (((_la-38)&-(0x1f+1)) == 0 && ((1<<uint((_la-38)))&((1<<(grulev3ParserSIMPLENAME-38))|(1<<(grulev3ParserDQUOTA_STRING-38))|(1<<(grulev3ParserSQUOTA_STRING-38))|(1<<(grulev3ParserDECIMAL_FLOAT_LIT-38))|(1<<(grulev3ParserHEX_FLOAT_LIT-38))|(1<<(grulev3ParserDEC_LIT-38))|(1<<(grulev3ParserHEX_LIT-38))|(1<<(grulev3ParserOCT_LIT-38)))) != 0) {
```

Comment:
Integer overflow on ARM

Reply:
Thanks for the review, sure will do that